source/FreeRTOS_ARP.c

```diff
@@ -290,7 +290,7 @@ eFrameProcessingResult_t eARPProcessPacket( const NetworkBufferDescriptor_t * px
                 {
                     case ipARP_REQUEST:
 
-                        if( ( ulTargetProtocolAddress == pxTargetEndPoint->ipv4_settings.ulIPAddress ) &&
+                        if( ( ulTargetProtocolAddress == pxTargetEndPoint->ipv4_settings.ulIPAddress ) && /* This check seems unnecessary which makes the first if condition in Line 356 obsolete. */
```

Comment:
I don't get it, it seems like IP address is possible to be different.

Reply:
updated

Reply:
We probably can remove the comment here.